### PR TITLE
fix(print): fixes tailwind/nuxt + e2e tests for print

### PR DESCRIPTION
**Pull request by usu**
Nuxt 2.16 introduced some breaking changes around PostCSS, so tailwind was currently not properly loaded for nuxt print.

This PR fixes these breaking changes and introduces end-to-end tests for both Nuxt and React print.

A further improvement could be to verify the PDF content after download (currently ist just verifies that a pdf file is downloaded).

### CODE REVIEW

**Comment by BacLuc** [comment]
Sorry @usu for https://github.com/ecamp/ecamp3/pull/3275/commits/db0c8ef61d00aaa18bae629d97c0b6b03838ca0b

**Review by BacLuc — APPROVED** [review]

File: frontend/cypress.config.js [hunk 1]
```diff
@@ -13,5 +13,10 @@ module.exports = defineConfig({
     },
     specPattern: 'tests/e2e/specs/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: 'tests/e2e/support/index.js',
+    baseUrl: 'http://localhost:3000',
```
Comment (carlobeltrame) [inline frontend/cypress.config.js:16]: This will need to be adjusted in the subpath PRs of @BacLuc

**Review by carlobeltrame — APPROVED** [review]

**Comment by usu** [comment]
> Sorry @usu for [db0c8ef](https://github.com/ecamp/ecamp3/commit/db0c8ef61d00aaa18bae629d97c0b6b03838ca0b)

Oh, what the f*. Thank, this drove me nuts.
How did you find out, permissions was the problem?

**Comment by BacLuc** [comment]
> > Sorry @usu for [db0c8ef](https://github.com/ecamp/ecamp3/commit/db0c8ef61d00aaa18bae629d97c0b6b03838ca0b)
> 
> Oh, what the f*. Thank, this drove me nuts. How did you find out, permissions was the problem?

You can enable debug logs when rerunning a check.
There you see it: https://github.com/ecamp/ecamp3/actions/runs/4217647599/jobs/7329848209

**Pull request merged**